src/clusterfuzz/_internal/bot/tasks/utasks/fuzz_task.py

```diff
@@ -859,10 +859,10 @@ def postprocess_process_crashes(uworker_input: uworker_msg_pb2.Input,
     # Artificial delay to throttle appengine updates.
     time.sleep(1)
 
-  # TODO(metzman): Replace fuzz_task_output.fully_qualified_fuzzer_name` with
-  # `fuzz_task_input.fuzz_target` instead.
-  upload_job_run_stats(fuzz_task_output.fully_qualified_fuzzer_name,
-                       uworker_input.job_type, fuzz_task_output.crash_revision,
+  # If there is no fuzz_target, this will be None.
+  stats_fuzzer_name = fuzz_target or fully_qualified_fuzzer_name
```

Comment:
Hmm, I think the original problem was that somehow these lines above didn't work:

```
if uworker_input.fuzz_task_input.HasField('fuzz_target'):
    fuzz_target = uworker_io.entity_from_protobuf(
        uworker_input.fuzz_task_input.fuzz_target, data_types.FuzzTarget)
    fully_qualified_fuzzer_name = fuzz_target.fully_qualified_name()
```

Leading to `fully_qualified_fuzzer_name` defaulting to the engine name.